Mihhail tsaplogin

theelkhound opened the pull request:
user page created

theelkhound committed:
first commit for user

theelkhound committed:
#31
implemented UserProfile view, implemented user.html

aeciovc commented on myquote/accounts/views.py:
```diff
@@ -51,3 +53,11 @@ def user_signup(request):
+    template_name = 'user.html'
+    context_object_name = 'user'
+    fields = ['first_name', 'last_name', 'email']
+    success_url = 'http://127.0.0.1:8000'
```
Try to use `reverse_lazy` function.

theelkhound replied:
done

theelkhound committed:
#31
upgraded UpdateProfile view,
added proper css for user.html,
linked user profile in the menu with proper page

theelkhound committed:
#31
updated user.html with draft user profile form working properly
